chirho/dynamical/handlers/solver.py

```diff
@@ -1,7 +1,16 @@
+from __future__ import annotations
+
+from typing import TypeVar
+
+import torch
+
 from chirho.dynamical.internals.solver import Solver
 
+S = TypeVar("S")
```

Comment:
Looks like these TypeVars aren't used anymore.

Reply:
There are lots of those in the codebase, but neither `mypy` nor `black`/`flake8` seem to mind so I just let them be.